### ISSUE TITLE
[BUG] Download not starting on Steam Deck

### ISSUE THREAD

**Issue by Pipodi** [hydralauncher/hydra#1178]
### Describe the bug

When I select a download from the popup, it goes in "Downloads" tab but it doesn't start

### Steps to Reproduce

1. Select a game
2. Select a download version
3. The game goes into the Download tab
4. The download doesn't start

### Expected behavior

The download starts

### Additional information and data

[error.txt](https://github.com/user-attachments/files/17587894/error.txt)
[info.txt](https://github.com/user-attachments/files/17587896/info.txt)
[pythoninstance.txt](https://github.com/user-attachments/files/17587897/pythoninstance.txt)

### Operating System

Steam Deck

### Hydra Version

3.0.4

### Before opening this Issue

- [X] I have searched the issues of this repository and believe that this is not a duplicate.
- [X] I am aware that Hydra team does not offer any support or help regarding the downloaded games.
- [X] I have read the [Frequently Asked Questions (FAQ)](https://github.com/hydralauncher/hydra/wiki/FAQ).

**Comment by Baregamyan** [comment]
Same problem on Ubuntu 24.04 with Hydra 3.0.4

**Comment by teteuhere** [comment]
Yes, on Ubuntu this problem started with 3.0.4, in 3.0.3 everything was fine.

**Comment by lmelo1994** [comment]
I have same problem on ZorinOS , on 3.0.3 is fine

**Comment by kataclank** [comment]
Same problem on steamdeck, this problem started with 3.0.4, in 3.0.3 everything was fine.

**Comment by zamitto** [comment]
We will release a new version with the fix ASAP

**Comment by zamitto** [comment]
Fixed in 3.0.5
